### PR TITLE
[node-manager] Add note about Manual approval mode for master

### PR DESCRIPTION
**Pull request by nabokihms**
Signed-off-by: m.nabokikh <maksim.nabokikh@flant.com>

## Description
Clarify nuances of changing approval mode for the master node group

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: node-manager
type: chore
summary: Add note about Manual approval mode for the master node group.
impact_level: low
```

### CODE REVIEW

File: candi/openapi/doc-ru-node_group.yaml [hunk 1]
```diff
@@ -245,7 +245,7 @@ spec:
                     approvalMode:
                       description: |
                         Режим выдачи разрешения на disruptive-обновление (обновление, требующее прерывание работы узла):
-                        - `Manual` — отключить автоматическую выдачу разрешений на disruptive-обновление. Если потребуется disruptive-обновление, то загорится специальный алерт.
+                        - `Manual` — отключить автоматическую выдачу разрешений на disruptive-обновление. Если потребуется disruptive-обновление, то загорится специальный алерт. **Внимание!** Для группе узлов `master` режим выдачи разрешения всегда должен быть Manual чтобы избежать проблем при выгоне (draining).
```
Comment (z9r5) [inline candi/openapi/doc-ru-node_group.yaml:248]:
```suggestion
                        - `Manual` — отключить автоматическую выдачу разрешений на disruptive-обновление. Если потребуется disruptive-обновление, то загорится специальный алерт. **Внимание!** Для группы узлов `master` режим выдачи разрешения всегда должен быть Manual чтобы избежать проблем при выгоне (draining).
```

File: candi/openapi/doc-ru-node_group.yaml [hunk 2]
```diff
@@ -478,7 +478,7 @@ spec:
...
```
Comment (z9r5) [inline candi/openapi/doc-ru-node_group.yaml:481]:
```suggestion
                        - `Manual` — отключить автоматическую выдачу разрешений на disruptive-обновление. Если потребуется disruptive-обновление, то загорится специальный алерт. **Внимание!** Для группы узлов `master` режим выдачи разрешения всегда должен быть Manual чтобы избежать проблем при выгоне (draining).
```

**Review by z9r5 — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [3153](https://github.com/deckhouse/deckhouse/pull/3153) to the branch [release-1.41](https://github.com/deckhouse/deckhouse/tree/release-1.41) successful!

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [3154](https://github.com/deckhouse/deckhouse/pull/3154) to the branch [release-1.40](https://github.com/deckhouse/deckhouse/tree/release-1.40) successful!

**Pull request merged**